working-with-multiple-clusters.md

```diff
@@ -0,0 +1,196 @@
+As a developer, there's going to be a time where you need to work with different Kubernetes clusters. At a minimum, you need to work with two clusters. One is your local clusters, and a second one could be a remote cluster for testing purposes in a cloud provider or elsewhere. Kubernetes has a set of configurations you can use to interact with different clusters from your workstation using kubectl. At the end of the day, remember that you're interacting with an API to persist the desired state for your workloads.
```

Comment:
```suggestion
As a developer, there's going to be a time where you need to work with different Kubernetes clusters. At a minimum, you need to work with two clusters: one is your local cluster, and a second one could be a remote cluster for testing purposes in a cloud provider or elsewhere. Kubernetes has a set of configurations you can use to interact with different clusters from your workstation using kubectl. At the end of the day, remember that you're interacting with an API to persist the desired state for your workloads.
```

I feel like we're missing a sentence or to briefly present contexts and explain how this solves the "two clusters" problem.